### PR TITLE
PLT-4429 disabling at_all at_channel metions mentions when channel has more than 1k users

### PR DESCRIPTION
**Pull request by coreyhulen**

### CODE REVIEW

**Comment by jwilander** [comment]
Some build errors:
```
api/post_test.go:1065: assignment count mismatch: 3 = 5
api/post_test.go:1074: assignment count mismatch: 3 = 5
api/post_test.go:1090: assignment count mismatch: 3 = 5
api/post_test.go:1096: assignment count mismatch: 3 = 5
api/post_test.go:1102: assignment count mismatch: 3 = 5
api/post_test.go:1109: assignment count mismatch: 3 = 5
api/post_test.go:1116: assignment count mismatch: 3 = 5
api/post_test.go:1122: assignment count mismatch: 3 = 5
api/post_test.go:1129: assignment count mismatch: 3 = 5
api/post_test.go:1136: assignment count mismatch: 3 = 5
api/post_test.go:1136: too many errors
FAIL	github.com/mattermost/platform/api [build faile
```

**Comment by hmhealey** [comment]
Should this number be configurable? As it is, this'll prevent explicit mentions from firing in Reception on Pre-release

**Comment by coreyhulen** [comment]
@hmhealey where is it preventing explicit mentions?  it should only be blocking @all, @channel, and @here.

**Comment by enahum** [comment]
I do agree with @hmhealey for me the `MAX_USERS_IN_CHANNEL_FOR_EXTRA_WORK` should be configurable somewhere

**Comment by coreyhulen** [comment]
I'll add it to the admin console.

**Comment by coreyhulen** [comment]
@enahum and @hmhealey I've added the setting to the system console.

File: i18n/en.json [hunk 1]
```diff
@@ -3333,6 +3333,10 @@
   },
   {
     "id": "model.config.is_valid.max_channels.app_error",
+    "translation": "Invalid maximum notifications per channel for team settings.  Must be a positive number."
+  },
+  {
+    "id": "model.config.is_valid.max_notify_per_channel.app_error",
```
Comment (hmhealey) [inline i18n/en.json]: These translations are for the wrong strings

**Comment by coreyhulen** [comment]
@hmhealey fix the translation string problem. ready for re-review.

**Pull request merged**